### PR TITLE
20.04 preperations

### PR DESCRIPTION
**Pull request by enoch85**
TODO:

- [ ]  Deprecate Ubuntu 16.04 from all scripts
- [x] Fix Nextcloud update script
- [ ]  Change base VMs to US keyboard
- [ ]  Update documentation (activate-ssl.sh, Swedish keyboard)
- [x]  Don't allow PHP lower than 7.4 (and 7.2 for a while)
- [x]  Deprecate MySQL/MariaDB
- [ ] ...

### CODE REVIEW

File: lets-encrypt/activate-tls.sh [hunk 1]
```diff
@@ -56,7 +56,7 @@ then
     sleep 1
 else
 msg_box "OK, but if you want to run this script later,
-just type: sudo bash /var/scripts/activate-ssl.sh"
+just type: sudo bash /var/scripts/activate-rls.sh"
```
Comment (szaimen) [inline lets-encrypt/activate-tls.sh:59]: typo?

File: nextcloud_install_production.sh [hunk 2]
```diff
@@ -286,7 +260,7 @@ check_command apt install -y \
     php"$PHPVER"-zip \
     php"$PHPVER"-mbstring \
     php"$PHPVER"-soap \
-    php"$PHPVER"-smbclient \
+#    php"$PHPVER"-smbclient \
```
Comment (szaimen) [inline nextcloud_install_production.sh:263]: is this not working on 20.04?

File: nextcloud_install_production.sh [hunk 3]
```diff
@@ -241,7 +215,7 @@ CREATE USER $NCUSER WITH PASSWORD '$PGDB_PASS';
 CREATE DATABASE nextcloud_db WITH OWNER $NCUSER TEMPLATE template0 ENCODING 'UTF8';
 END
 print_text_in_color "$ICyan" "PostgreSQL password: $PGDB_PASS"
-service postgresql restart
+systemctl restart postgresql
```
Comment (szaimen) [inline nextcloud_install_production.sh:218]: is here a postgresql.service missing? you have done it this way below.

File: nextcloud-startup-script.sh [hunk 4]
```diff
@@ -155,18 +149,18 @@ debug_mode
 lowest_compatible_nc 16
 
 # Check that this run on the PostgreSQL VM
-if ! is_this_installed postgresql-10
+if ! is_this_installed postgresql-common
```
Comment (szaimen) [inline nextcloud-startup-script.sh:152]: is there no longer a specific version on 20.04?

File: nextcloud_update.sh [hunk 5]
```diff
@@ -160,7 +160,7 @@ if version 18.04 "$DISTRO" 18.04.10; then
         fi
         pecl channel-update pecl.php.net
         yes no | pecl upgrade redis
-        service redis-server restart
+        systemctl restart redis-server
```
Comment (szaimen) [inline nextcloud_update.sh:163]: shouldn't it be redis-server.service?

File: lets-encrypt/activate-tls.sh (hunk 1 above)
Comment (enoch85) [reply lets-encrypt/activate-tls.sh:59]: thanks!

File: nextcloud_install_production.sh (hunk 2 above)
Comment (enoch85) [reply nextcloud_install_production.sh:263]: nope, not yet anyway.

File: nextcloud_install_production.sh (hunk 3 above)
Comment (enoch85) [reply nextcloud_install_production.sh:218]: Yes, I will fix all the details later, thanks for pointing it out. :)

File: nextcloud-startup-script.sh (hunk 4 above)
Comment (enoch85) [reply nextcloud-startup-script.sh:152]: It's 12, but this is more generic.

File: nextcloud_update.sh (hunk 5 above)
Comment (enoch85) [reply nextcloud_update.sh:124]: WIll check later, thanks!

**Pull request closed without merging**